packages/apps/tools/src/utils/network.ts

```diff
@@ -61,19 +56,21 @@ export const getAllNetworks = (
       }),
     );
   }
-  return allNetworkObjects.filter((item) => item.networkId !== 'testnet05');
+  return allNetworkObjects.filter(
+    (item) => (item.networkId as string) !== 'testnet05',
+  );
 };
```

Comment:
Add a comment here to as why we do this
```suggestion
  }
	// remove leftover `testnet05` from the localStorage to clean up the users data
  return allNetworkObjects.filter(
    (item) => (item.networkId as string) !== 'testnet05',
  );
};
```